### ISSUE TITLE
aria-valid-attr rule incorrectly prohibits <div> tags from having aria-placeholder attribute

### ISSUE THREAD

**Issue by jackjlynch** [dequelabs/axe-core#1457]
Expectation: Axe should accept `aria-placeholder` as a valid attribute on any element that can't have a `placeholder` attribute, i.e. everything except for `input` and `textarea`.

Actual: When run on the following html:
````html
<div aria-placeholder="placeholder text"></div>
````
Axe returns this violation:
````
    "ruleDescription": "Ensures attributes that begin with aria- are valid ARIA attributes",
    "helpUrl": "https://dequeuniversity.com/rules/axe/3.0/aria-valid-attr?application=axeAPI",
    "ruleId": "aria-valid-attr",
    "impact": "critical",
````
Motivation: This use of `aria-placeholder` conforms to the spec and should not cause a violation.

axe-core version: 3.0.0-alpha-1

**Comment by WilcoFiers** [comment]
@jackjlynch `aria-placeholder` is only supported by elements with a `textbox` or `searchbox` role. If you put either of those roles on the div, axe will pass.

I just had a quick test of this, and it doesn't seem to work, at least not in VoiceOver & Safari. Unless you add the role to it, the placeholder text is ignored.

**Comment by jackjlynch** [comment]
Thanks for the correction, I must have misread the spec. I'm still able to reproduce this in our testing environment even with `role=textbox`, but not when I load axe in the browser. Not sure what's causing the problem yet but it's looking less like a bug in axe at this point.
